dbbackup/management/commands/mediabackup.py

```diff
@@ -86,14 +86,14 @@ def backup_mediafiles(self):
                                            servername=self.servername,
                                            content_type=self.content_type)
 
-        outputfile = self._create_tar(filename)
+        tarball = self._create_tar(filename)
```

Comment:
I thought it's destination archive file, but it was source tarball. Naming is hard 😩